egs/spgispeech/s5/path.sh

```diff
@@ -0,0 +1,10 @@
+export KALDI_ROOT=$PWD/../../..
+export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PWD:$PATH
+[ ! -f $KALDI_ROOT/tools/config/common_path.sh ] && echo >&2 "The standard file $KALDI_ROOT/tools/config/common_path.sh is not present -> Exit!" && exit 1
+. $KALDI_ROOT/tools/config/common_path.sh
+. $KALDI_ROOT/tools/env.sh
+conda activate k2 || true
```

Comment:
Maybe we can mention in the README that this recipe depends on Lhotse for data preparation.